package.json

```diff
@@ -50,6 +50,7 @@
     "hast-util-to-html": "^9.0.0",
     "hast-util-to-jsx-runtime": "^2.3.0",
     "hast-util-to-string": "^3.0.0",
+    "i18next": "^23.7.20",
```

Comment:
i actually think because we do i18n at compile time, we can get away with not using an extra library here at all and just simply do a lookup into that resources object that you created with some string interpolation

each file will probably be something like

```ts
const enTranslations = {
  seeRemaining: ({remaining}) => `See {remaining} more`
}
```

most i18n libraries are designed for client-side dynamic i18n so I think we can do better here

Reply:
I use i18next because I think it make easy to maintains, specially under VSCode. More over, using crowdin, we could also support to translate in multiple language if we use JSON file instead of ts file for translation.

Maybe we could make something that use a local function to translate `{{variable}}` without using i18next ?

Reply:
Also, I don't know how I can build the default main language without using i18next :<, as some component doesn't load config file

Reply:
About the i18n function, maybe we could tweaks around types to allow at last autocompletion ?
The problem is : if we add new key, we must edit manually the two JSON + know the key set.

Using i18n prevent this problem as VSC already know and the plugin i18n works for that.